docs/concept-exercises.md

```diff
@@ -51,30 +51,51 @@ An exercise has the following files. In the browser they will show at the releva
+**Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
-As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes". Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
+- The information provided should give the student just enough context to figure out the solution themselves. 
+- It should not give the student any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
+- Use the proper technical terms so that the student can easily search for more information. 
+- If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+
+As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes", tell the users how to create a string, and explain that a string has methods that can be used to manipulate it. Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.
 
 See the C# floating-point-numbers exercise's [introduction.md file][csharp-docs-introduction.md] for an example.
 
 ### `.docs/instructions.md`
 
-This file contains instructions for the exercise. It should explicitly explain what the student needs to do (define a method with the signature `X(...)` that takes an A and returns a Z), and provide at least one example usage of that function. If there are multiple tasks within the exercise, it should provide an example of each.
+**Purpose:** Provide instructions for the exercise.
+
+This file is split into two halves. 
+1. The first half explains the "story" or "theme" of the exericise. It should generally contain no code samples.
+2. The second half provides clear instructions of what a student needs to do, providing tasks.
+
+Each tasks must conform to the following standard:
+- Start with a third-level heading starting with a number (.e.g `### 1.`, `### 2.`)
+- Provide a method that a student needs to define with its signature(e.g. `X(...)` that takes an A and returns a Z),
+- Provide at example usage of that function. These examples should be different examples to those given in the tests.
 
-See the C# floating-point-numbers exercise's [instructions.md file][csharp-docs-instructions.md] for an example.
+Avoid any controversial topics in the stories - be empathetic to triggers that people might have - there's normally a less controversial or risque variant of a story that can be used.
```

Comment:
Could you define "controversial"?

Reply:
I am also curious too but don't think we need to delay merging the PR on a definition. I would interpret this by Exercism's [code of conduct](https://exercism.io/code-of-conduct).

Reply:
Anything that might offend anyone. In one example exercise there was a casual reference to Covid-19, which while not in itself malicious or nasty, could easily be triggering by someone who lost a friend to it. I've clarified in a suggestion below.

Reply:
> but don't think we need to delay merging the PR on a definition

I think it should either be left out of this PR or clarified. The CoC applies anyway and this goes beyond it and has the potential to violate the spirit of the CoC itself.

If you're not a native speaker (or in many cases not directly familiar with US-American culture), it can be incredibly hard to figure out what you're "allowed" to say or not. There has been one recent case in problem-specs where someone wanted to change phrasing of some examples because they considered it too offensive or vulgar even though it wasn't a violation of the CoC. Having such a vague requirement in this doc can scare people away from writing exercises in the first place, especially when there have been examples where something was "controversial" beyond the way the CoC specifies it. Scaring people away from writing exercises is generally not something we want to achieve, I think.

Instead of a negative requirement, some encouragement would be better if you want to leave an additional hint in this doc. Something like "In the spirit of the CoC, we want to keep exercise stories welcoming to everyone, here are some things you could include to ensure that: <list of suggestions>" or "Please keep the CoC in mind when writing the story for your exercise".